src/server/src/main/java/io/cassandrareaper/core/Table.java

```diff
@@ -21,12 +21,22 @@
 
 public final class Table {
 
+  /**
+   * Default compaction strategy, used when the strategy cannot be retrieved from JMX. The
+   * information is only available since Cassandra 2.1
+   */
+  private static final String DEFAULT_COMPACTION_STRATEGY = "SizeTieredCompactionStrategy";
```

Comment:
@rborer would/does the code break if we make the default value `"unknown"`? i'm hesitant about presuming a compaction strategy, as we may take an action based on that and it would not be accurate against Cassandra <2.1

Reply:
Yeah I've been wondering about putting a default value that doesn't exists here. As far as I can tell the compaction strategy information is only used to skip tables using Time Window / Date Tiered data. So it should be safe to put "unknown" in there instead.